src/main/java/org/broadinstitute/consent/http/exceptions/ConsentConflictException.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.consent.http.exceptions;
+
+public class ConsentConflictException extends Exception{
+  public ConsentConflictException() {
+    super();
+  }
+
+  public ConsentConflictException(String message) {
+    super(message);
+  }
+
+  public ConsentConflictException(String message, Throwable cause) {
+    super(message, cause);
+  }
+
+  public ConsentConflictException(Throwable cause) {
+    super(cause);
+  }
+
+  public ConsentConflictException(String message, Throwable cause, boolean enableSuppresion, boolean writableStackTrace) {
+    super(message, cause, enableSuppresion, writableStackTrace);
+  }
+}
```

Comment:
I think this would be more accurate if it extended ClientErrorException. This would be more than enough of a class:
```
public class ConsentConflictException extends ClientErrorException {
  public ConsentConflictException() {
    super(Response.Status.CONFLICT);
  }
}
```